practices/images-best-practices.md

```diff
@@ -0,0 +1,48 @@
+# Best practices when dealing with images for the Web
+
+In a front-end workflow we typically deal with two kinds of images:
+
+* **Icons, logos, buttons and other graphical elements** that are part of the User Interface of the page. These are usually managed by the frontend, and so they are stored with the rest of resources (templates, CSS, JS). Like those resources, they may be shared between many different pages.
+
+* **Content images**. These are unique to a page (or a handful of pages). They will be usually provided from a database or a content management system.
+
+Most best practices are common to both categories.
+
+## Formats
+
+### What format to use
+
+* Use SVGs whenever you want to display a vector image.
+* Use JPEGs for photo images, or images with many little details.
+* Use PNGs for graphs and other images with large areas of the same colour. Also for images that require transparency.
+* Don't use GIFs.
+
+### About WebP
+
+We don't encourage the use of the [WebP format](https://developers.google.com/speed/webp/) as support for it is still not widespread. [MS is not planning to add support to Edge](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/webpimageformat/) at the moment. Firefox is currently working in adding support for it, so we may reconsider after that.
```

Comment:
Does 'not encouraging' something mean to discourage it?

Anyway, if global support for webp is [over 70%](http://caniuse.com/#search=webp) - not sure if this is a good recommendation since the appropriate image format can be served to supporting clients via content negotiation.